lib/chibi/snow/utils.scm

```diff
@@ -25,6 +25,10 @@
             ,(delay
                (process->sexp
                 '(foment -e "(write (features))"))))
+    (generic "generic" #f #f
+            ,(delay
+               (process->sexp
+                '(echo "generic"))))
```

Comment:
I guess we can just `(write-string "generic\n")` instead of using a sub-process.